pkg/services/fake/catalog.go

```diff
@@ -40,6 +40,15 @@ func (m *Module) GetCatalog() (service.Catalog, error) {
 						"Fake 2",
 					},
 				},
+				Schemas: service.PlanSchemas{
+					ServiceInstances: service.InstanceSchemas{
+						ProvisioningParametersSchema: service.InputParametersSchema{
+							PropertySchemas: map[string]service.PropertySchema{
+								"someParameter": &service.StringPropertySchema{},
+							},
+						},
+					},
+				},
```

Comment:
Validation now rejects fields not in the schemas. Functional tests against the `api` package often use the parameter `someParameter`. Those were all failing because they weren't in the schema for this fake plan-- so I added schema for the fake plan.